src/main/java/org/apache/sling/jcr/packageinit/impl/ExecutionPlanRepoInitializer.java

```diff
@@ -132,6 +132,8 @@ public void processRepository(SlingRepository slingRepository) throws Exception
                 logger.info("Waiting for PackageRegistry.");
                 PackageRegistry registry = (PackageRegistry) st.waitForService(0);
                 logger.info("PackageRegistry found - starting execution of execution plan");
+                // workaround until https://issues.apache.org/jira/browse/JCRVLT-517 is solved: registry.contains yields wrong value until initialized
```

Comment:
Can you add a hint, where registry.contains() is called as this doesn't happen in this class...

Reply:
In the code comment? OK, done.